cypress.config.js

```diff
@@ -0,0 +1,11 @@
+const { defineConfig } = require("cypress");
+
+module.exports = defineConfig({
+  e2e: {
+    baseUrl: 'http://localhost:3000',
+    supportFile: false
+  },
+  env: {
+    API_URL: 'http://localhost:3001',
+  }
+  })
```

Comment:
```suggestion
})
```

Cuidado com a indentação!